tests/meta/backup/test_meta_backup.py

```diff
@@ -78,7 +76,7 @@ def test_get_first_flow_cell_next_requested(mock_store, mock_flow_cell):
     # WHEN a flow cell is requested to be retrieved from PDC
     mock_store.get_flow_cells_by_statuses(
         flow_cell_statuses=[FlowCellStatus.REQUESTED]
-    ).first.return_value = mock_flow_cell
+    ).return_value = mock_flow_cell
```

Comment:
The mocking here looks strange. Are mock_store and mock_flow_cell not mixed up?

Reply:
The return should be a list of flow cell indeed.